### DISCUSSION TITLE
pcl的语言是jvav的还是c#的

### DISCUSSION THREAD

**Discussion by Xmk114QwQ** [Meloong-Git/PCL#384] [提问] [answered]
pcl的代码时jvav的还是c#的，我怎么改

**Comment by LTCatt** [comment, marked as answer]
VB.net，等效 C#

**Reply by Xmk114QwQ** [reply]
谢谢啦，当时没看到，这个语言我不太会。本来还想着反编译（雾）

**Comment by KOFTWAR** [comment]
https://github.com/Hex-Dragon/PCL2/
这是PCL2的源代码
你向下翻，右下角有“Languages”
在里面可以看到语言，是Visual Basic .NET
